src/applications/vaos/services/mocks/var/requests.json

```diff
@@ -967,10 +967,10 @@
           }
         ],
         "lastUpdatedAt": null,
-        "createdDate": "07/14/2020 15:27:42",
+        "createdDate": "01/14/2021 15:27:42",
```

Comment:
We use three different date fields for EC, this updates the mocks to keep all those dates in sync.